src/main/java/hu/bme/mit/spaceship/GT4500.java

```diff
@@ -21,8 +16,10 @@ public GT4500() {
   }
 
   public boolean fireLaser(FiringMode firingMode) {
-    // TODO not implemented yet
-    return false;
+   	if(1 < 2)
+   	 return false;
```

Comment:
this implementation does not meet the specifications requirements

Reply:
:(

Reply:
just do it